urdf/package.xml

```diff
@@ -24,12 +24,12 @@
   <build_depend>console_bridge</build_depend>
   <build_depend>urdfdom</build_depend>
   <build_depend>urdfdom_headers</build_depend>
-  <build_depend>TinyXML</build_depend>
+  <build_depend>tinyxml</build_depend>
   <build_depend>tinyxml_vendor</build_depend>
 
   <exec_depend>urdfdom</exec_depend>
-  <exec_depend>urdfdom-headers</exec_depend>
-  <exec_depend>TinyXML</exec_depend>
+  <exec_depend>urdfdom_headers</exec_depend>
```

Comment:
Looks like the tinyxml dependency has been introduced when porting because it was before provided by another package that has been striped. I think that `tinyxml` is fine as it and that using `TinyXML` was an oversight by using the "cmake package name" rather than the rosdep key